manifests/init.pp

```diff
@@ -188,6 +191,7 @@
   Stdlib::Fqdn $servername = $facts['networking']['fqdn'],
   Array[Stdlib::Absolutepath] $allowed_import_path = ['/var/lib/pulp/sync_imports'],
   Array[Stdlib::Absolutepath] $allowed_export_path = [],
+  Array[Enum['md5', 'sha1', 'sha224', 'sha256', 'sha384', 'sha512']] $allowed_content_checksums = ['sha224', 'sha256', 'sha384', 'sha512'],
```

Comment:
I did suggest to extract the `Array[Enum[]]` to a type alias. That makes it easy to reuse in another module.

Reply:
ahh, thanks for clarifying, will add that

Reply:
I should have linked back to https://github.com/theforeman/puppet-pulpcore/pull/183#discussion_r615239844 for that suggestion.

Reply:
added!